.maintain/rustdocs-release.sh

```diff
@@ -0,0 +1,244 @@
+#!/usr/bin/env bash
+# set -x
+
+# This script manages the deployment of Substrate rustdocs to https://paritytech.github.io/substrate/.
+# - With `deploy` sub-command, it will checkout the passed-in branch/tag ref, build the rustdocs
+#   locally (this takes some time), update the `index.html` index page, and push it to remote
+#   `gh-pages` branch. So users running this command need to have write access to the remote
+#   `gh-pages` branch. This sub-command depends on [@jimmychu0807/index-tpl-crud](https://www.npmjs.com/package/@jimmychu0807/index-tpl-crud)
+#   to update the DOM of index.html page.
+# - With `remove` sub-command, it will remove the deployed rustdocs from `gh-pages`, and update the
+#   index.html page as necessary. It may remove the `latest` symbolic link.
+#
+# Examples:
+#   # Showing help text
+#   rustdocs-release.sh -h
+#
+#   # Deploy rustdocs of `monthly-2021-10` tag
+#   rustdocs-release.sh deploy monthly-2021-10
+#
+#   # In addition to the above, the `latest` symlink will point to this version of rustdocs
+#   rustdocs-release.sh deploy -l monthly-2021-10
+#
+#   # Remove the rustdocs of `monthly-2021-10` from `gh-pages`.
+#   rustdocs-release.sh remove monthly-2021-10
+#
+# Dependencies:
+#   - @jimmychu0807/index-tpl-crud - https://www.npmjs.com/package/@jimmychu0807/index-tpl-crud
+#
+
+# Script setting
+# The git repo http URL
+REMOTE_REPO="https://github.com/paritytech/substrate.git"
```

Comment:
is it used somewhere? If it is externally, then please `export`

Reply:
No, just some kind of constant set at the beginning of the script.